src/Nethermind/Nethermind.Core/Eip4844Constants.cs

```diff
@@ -5,15 +5,42 @@
 
 namespace Nethermind.Core;
 
+/// <summary>
+/// Represents the <see href="https://eips.ethereum.org/EIPS/eip-4844#parameters">EIP-4844</see> parameters.
+/// </summary>
 public class Eip4844Constants
 {
+    /// <summary>
+    /// Gets the <c>GAS_PER_BLOB</c> parameter.
+    /// </summary>
+    /// <remarks>Defaults to 2e17.</remarks>
+    public const ulong GasPerBlob = 1 << 17;
+
     public const int MinBlobsPerTransaction = 1;
 
-    public const ulong BlobGasPerBlob = 1 << 17;
-    public const ulong TargetBlobGasPerBlock = BlobGasPerBlob * 3;
-    public const ulong MaxBlobGasPerBlock = BlobGasPerBlob * 6;
-    public const ulong MaxBlobGasPerTransaction = MaxBlobGasPerBlock;
+    /// <summary>
+    /// Gets the <c>BLOB_GASPRICE_UPDATE_FRACTION</c> parameter.
+    /// </summary>
+    /// <remarks>Defaults to 3338477.</remarks>
+    public static UInt256 BlobGasPriceUpdateFraction { get; set; } = 3338477;
```

Comment:
@flcl42 Do we really need this to be `UInt256` instead of `ulong`?

Reply:
It's used in an operation that requires UInt256 input, you can make it here ulong, but I see no big difference.